api/graphql/makeModels.js

```diff
@@ -213,7 +213,7 @@ export default function makeModels (userId, isAdmin) {
       model: Vote,
       attributes: [
         'id',
-        'created_at'
+        'date_voted'
```

Comment:
Let's do a `createdAt` getter that relies on `date_voted` so that the API's outer surface is consistent